src/Analyzer.BicepProcessor/BicepTemplateProcessor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.Azure.Templates.Analyzer.TemplateProcessor;
```

Comment:
Add copyright header to the top of file

``` c#
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```

Reply:
Move the System using statement to the top of the using statements and then alphabetize the remaining below it.

Reply:
`Microsoft.Azure.Templates.Analyzer.TemplateProcessor` is not used so you can remove it